system/emergency_handler/src/emergency_handler/emergency_handler_core.cpp

```diff
@@ -305,7 +305,10 @@ EmergencyHandler::selectAlternativeControlCommand()
   // Emergency Stop
   {
     autoware_auto_control_msgs::msg::AckermannControlCommand emergency_stop_cmd;
+    emergency_stop_cmd.stamp = this->now();
```

Comment:
@shmpwk 
this time stamp is missing so please keep this fix but please remove commit overwriting time stamp

Reply:
Ok, I add stamp only when emergency stop cmd stamp is empty with warning.